components/card/card.css

```diff
@@ -0,0 +1,241 @@
+/* @import url('../../components/root.css');
+@import url('../../components/Buttons/button.css');
+@import url('../../components/shadow.css'); */
+
+
+@import url('../../components/framer.css');
+
+/* card-close button*/
+.card-close-btn{
+    padding: 4px;
+    margin: 0px;
+    border-radius: 50%;
+    height: 3%;
+    cursor: pointer;
+    border: 1px solid rgba(121, 121, 121, 0.0);
```

Comment:
Border opacity mentioned is 0. Instead, you can directly write "border: none"

Reply:
> Border opacity mentioned is 0. Instead, you can directly write "border: none"

This is related to border opacity not with border none 😁